README.md

```diff
@@ -48,6 +48,12 @@ Next.js Middleware works out of the box on Netlify. By default, middleware runs
 support for running Middleware at the origin, set the environment variable `NEXT_DISABLE_NETLIFY_EDGE` to `true`. Be
 aware that this will result in slower performance, as all pages that match middleware must use SSR.
 
+### Limitations
+
+Due to how the site configuration is handled when it's run using Netlify Edge Functions, data such as `locale` and `defaultLocale` will be missing on the `req.nextUrl` object when running `netlify dev`. 
```

Comment:
When you say "such as", does that mean there are other things missing?

Reply:
I haven't yet done an audit of all the properties that are affected by this so I'm not 100% certain that these are the only properties affected on the `nextUrl` object.

Something to add here as well but the pathname (as reported by the user who originally raised this issue with us) was also incorrect before these changes and should be mentioned.